tests/unit/challenges/test_views.py

```diff
@@ -127,6 +128,11 @@ def test_create_challenge_with_no_data(self):
         response = self.client.post(self.url, self.data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_check_challenge_host_team_ownership(self):
+        del self.date['team_name']
+        response = self.client.post(self.url, self.data)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)    
```

Comment:
@gauthamzz see this https://travis-ci.org/Cloud-CV/EvalAI/builds/189953817#L776

There is a whitespace at the end of this line. Please remove it and it should work fine.